search/text_search.cpp

```diff
@@ -12,31 +12,65 @@
 
 /** Main function
  */
+
+std::string lower(std::string word) {
+    int length = word.length();

```

Comment:
Please rever the changes in this file or make a separate PR for this.

Reply:
I'm really sorry for this mistake. I'll make a new PR right away 😃